cypress/integration/26-register_publication/417-creator-navigates-to-contributor-tab/contributors.js

```diff
@@ -0,0 +1,46 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_WITH_AUTHOR } from '../../../support/constants';
+import 'cypress-localstorage-commands';
+import 'cypress-file-upload';
+
+const testFile = 'example.txt';
+
+Given('Creator begins registering a Registration in the Wizard', () => {
+  cy.loginCognito(USER_WITH_AUTHOR).then((idToken) => {
+    cy.wrap(idToken).as('idToken');
+    cy.setLocalStorage('i18nextLng', 'eng');
+    cy.setLocalStorage('previouslyLoggedIn', 'true');
+    cy.visit('/');
+    cy.get('[data-testid=new-publication]').click({ force: true });
+    cy.get('[data-testid=new-publication-file]').click({ force: true });
+    cy.get('input[type=file]').attachFile(testFile);
+  });
+});

```

Comment:
Ser vi har dette flere plasser nå, kan vi samle ting vi må gjøre flere plasser en annen plass (`common.js` e.l.), så kan vi bruke det overalt hvor det trengs:
```suggestion
Given('Creator begins registering a Registration in the Wizard', startRegistrationWithFile);
```

Reply:
Skal lage en task på det.

Reply:
ser ut som vi definerer den samme Given 5 ganger bare i denne PRen nå

Reply:
Det er mulig å dra den ut i `cypress/integration/common/`, men tror kanskje det da blir vanskeligere å skjønne testene.

Reply:
tror også det er lurt å trekke den ut. spesielt mtp at innholdet inni der kan endre seg (data-testid har allerede gjort det), og da er det mye bedre å ha det på kun en plass

Reply:
Tror jeg skal trekke det ut i `command.js` for å gjøre det lettere å lese tester etterpå. Kommer i en egen PR siden jeg vil fikse en del andre tester på samme måten.